Week1/homework/app.js

```diff
@@ -1,11 +1,197 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+  // This is an array that stores my books' information.
+  // Each element is an object that contains details.
+  const books = [
```

Comment:
This is supposed to be an object. You could use Object.keys() to convert it to an object on the fly and then apply .sort() array method

Reply:
I was using an object, then I wasn't sure how to use .sort() from an object, that was why I changed it directly to an array. But I've just fixed it with Object.keys(). So now it should be an object :) Thank's for the hint.